### PR TITLE
Synthetics Test Steps reorg

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
Reorganizes the Synthetics Test Steps page:

- Changes `actions` to `steps` to follow existing UI language
- Tweaks intro / step organization to let users know what steps are instantly recorded vs. manually need to be added
- Pulls assertions table out into individual sections for easier linking to customers

### Motivation
Jira request

### Preview link
https://docs-staging.datadoghq.com/sarina/browser-actions/synthetics/browser_tests/actions/

### CODE REVIEW

**Comment by sarina-dd** [comment]
@MargotLepizzera This is ready for review - I couldn't find the navigation table you were referencing in Jira, but let me know if we need to break down that section further.

**Comment by MargotLepizzera** [comment]
Amazing, thank you both! A few additional nits:

- Can this page be listed as `steps` in the sidebar as well? https://a.cl.ly/RBuvz9bW

- References to `actions` in the below pages should probably be changed to `steps` as well:
  - https://docs.datadoghq.com/synthetics/browser_tests/ 
  - https://docs.datadoghq.com/synthetics/browser_tests/advanced_options
  - https://docs.datadoghq.com/getting_started/synthetics/browser_test#record-your-test-steps

I think that should be mostly it, we can prob fix the rest if we spot other occurrences as we go!

- This paragraph https://docs.datadoghq.com/synthetics/browser_tests/#actions should match the new overview paragraph of https://docs.datadoghq.com/synthetics/browser_tests/actions/

- Regarding `navigation`, it's currently a list, if we could add subsections just like you did for `assertions` and like we have for `special actions` it would be great. I know these subsections feel a bit empty for now but I'll make sure to create a PR to add more details + links!

After that, it'd be good to merge on my end!

**Comment by sarina-dd** [comment]
Thanks for the info @MargotLepizzera, good catch on the side nav. I'll try to get to these changes today and will let you know when updated.

**Comment by sarina-dd** [comment]
Updated with suggestions 👍Regarding links, going to look into if we should update the url. I don't think it hurts to keep it as actions as the intro talks about steps as actions, but might be a bit confusing to some.

**Comment by MargotLepizzera** [comment]
Thanks!
 
- Agreed on the URL, if we can change it, I'm definitely in favor of switching it to steps as well.
- I took your original PR as an opportunity to perform several changes I have been meaning to work on for a while. Can you review https://github.com/DataDog/documentation/pull/7473 before merging yours?

File: content/en/synthetics/browser_tests/_index.md [hunk 1]
```diff
@@ -41,25 +41,30 @@ Define the configuration of your browser test.
...
-6. **How often should Datadog run the test?** Intervals are available between every 15 minutes to once per week. [Contact support][4] to enable additional frequencies for your test.
+4. **Devices**: The devices to run your test on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. You can also set up a [Private Location][2] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Intervals are available between every 5 minutes to once per week.
 
 ### Use global variables
 
-You can use the [global variables defined in the `Settings`][5] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
+You can use the [global variables defined in the `Settings`][3] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
```
Comment (sarina-dd) [inline content/en/synthetics/browser_tests/_index.md:50]:
```suggestion
You can use the [global variables defined in the **settings**][3] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
```

File: content/en/synthetics/browser_tests/_index.md [hunk 2]
```diff
@@ -41,25 +41,30 @@ Define the configuration of your browser test.
...
-6. **How often should Datadog run the test?** Intervals are available between every 15 minutes to once per week. [Contact support][4] to enable additional frequencies for your test.
+4. **Devices**: The devices to run your test on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. You can also set up a [Private Location][2] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Intervals are available between every 5 minutes to once per week.
 
 ### Use global variables
 
-You can use the [global variables defined in the `Settings`][5] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
+You can use the [global variables defined in the **settings**][3] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
```
Comment (sarina-dd) [inline content/en/synthetics/browser_tests/_index.md:50]:
```suggestion
You can use the [global variables defined in the **settings**][3] in the URL as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
```

File: content/en/synthetics/browser_tests/_index.md [hunk 3]
```diff
@@ -41,25 +41,30 @@ Define the configuration of your browser test.
...
+* Retry `<INSERT_NUMBER>` times before location is marked as failed. This allows you to define how many consecutive test failures need to happen for a location to be considered as failed. By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][4].
+
+{{< img src="synthetics/browser_tests/alerting_rules.png" alt="Browser test alerting rule"  >}}
 
 ### Notify your team
 
 A notification is sent according to the set of alerting conditions. To configure your notifications:
 
-1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][6] and supports the following [conditional variables][7]:
+1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][5] and supports the following [conditional variables][6]:
```
Comment (sarina-dd) [inline content/en/synthetics/browser_tests/_index.md:67]: @MargotLepizzera  Is this the correct link? Looks like it links out the integrations page, which fits with the content below in 2.

File: content/en/synthetics/browser_tests/actions.md [hunk 4]
```diff
@@ -26,57 +28,120 @@ Choose from:
...
+### Upload file
+
+You can record the uploading of files as a step. To record an **Upload** step you can:
+
+* Either open your desktop from the browser,
+* Or drag and drop your file in the recording iframe.
+
+{{< img src="synthetics/browser_tests/upload_file_step.png" alt="Create an upload file step"  style="width:60%;">}}
+
+This is limited to 10 files, with a limit of 5MB each.
```
Comment (sarina-dd) [inline content/en/synthetics/browser_tests/actions.md:52]:
```suggestion
This is limited to 10 files with a limit of 5MB each.
```

File: content/en/synthetics/browser_tests/_index.md (hunk 3 above)
Comment (MargotLepizzera) [reply content/en/synthetics/browser_tests/_index.md:67]: @sarina-dd great catch! This should link to monitor pages.

- conditional variables to https://docs.datadoghq.com/monitors/notifications/?tab=is_alert#integrations
- services to https://docs.datadoghq.com/monitors/notifications/?tab=is_alert#integrations

**Pull request merged**